### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer if appI…

### PR DESCRIPTION
**Pull request by OVikhrov**
…D of not registered app
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Not_Registered_App.lua [hunk 1]
```diff
@@ -0,0 +1,96 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Not_Registered_App.lua]: @OVikhrov, please update Requirement summary with at least one related functional requirement

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Not_Registered_App.lua [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_With_appID_Of_Not_Registered_App()
+ self.hmiConnection:SendNotification("UI.OnSeekMediaClockTimer",{seekTime = {hours = 30, minutes = 30, seconds = 30}, appID = self.applications["Test Application"]+1})
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Not_Registered_App.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Not_Registered_App.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Not_Registered_App.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Not_Registered_App.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Not_Registered_App.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**